core/Cargo.toml

```diff
@@ -20,7 +20,11 @@ include = ["/Cargo.toml", "src/**/*.rs", "/README.md", "/LICENSE"]
 [dependencies]
 ink_alloc = { path = "../alloc/" }
 ink_utils = { path = "../utils/" }
-parity-codec = { version = "3.2", default-features = false, features = ["derive", "full"] }
+parity-codec = { version = "4.1.1", default-features = false, features = ["derive", "full"] }
+
+[dev-dependencies]
+node-runtime = { git = "https://github.com/paritytech/substrate/", branch="aj-wasm-builder", package = "node-runtime", features = ["std"] }
```

Comment:
So here we are again ... depending on `node-runtime`. I guess it is only for tests, right? So does that mean that `ink_core` tests now compile forever? If so I would really appreciate this behind a crate feature and as an optional `dev-dependency` to be able to quickly run simple tests that do not rely on the `ext_dispatch_call` functionality.

Reply:
Now moved over to https://github.com/paritytech/ink-types-node-runtime/pull/3. But yes a good idea to have a feature to avoid compiling whole of substrate for these tests each time, and just have it run on CI.